toolkit/scripts/incremental_building.mk

```diff
@@ -62,6 +62,7 @@ DELTA_BUILD    = y
 USE_CCACHE    ?= y
 REBUILD_TOOLS ?= y
 DELTA_FETCH   ?= y
+PRECACHE	  ?= y
```

Comment:
nit: Convert tab to spaces to ensure whitespace is consistent.

Reply:
Forgot to push my final commit for this, should be fixed.